CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v2.5.0] - 2022-03-10
+### Changed
+
+- Split the `--recursive` option into a `--recursive` flag and a `--max-depth` option
+- Renamed the entry point from `stac_validator` to `stac-validator`
+
```

Comment:
What do you think about tagging this as 3.0 so people take note of the breaking changes?